### ISSUE TITLE
[indent] doesn't work correctly with decorators

### ISSUE THREAD

**Issue by dmitriybo** [typescript-eslint/typescript-eslint#4079]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

1. Install typeorm
2. Insert rule into your .eslintrc.json file
3. Insert code into new file

```JSON
{
  "rules": {
    "semi": [2, "never"],
    "indent": ["error", 2]
  }
}
```

```TS
import { Column } from 'typeorm'

export class BaseEntity {
  @Column({ type: 'boolean', default: true })
  isActive: boolean
}
```

**Expected Result**

This code must be marked typescript-eslint as correct

**Actual Result**

Eslint throws me this error (line 5): ESLint: Expected indentation of 4 spaces but found 2. (indent)

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.2.0` |
| `@typescript-eslint/parser`        | `5.2.0` |
| `TypeScript`                       | `4.4.4` |
| `ESLint`                           | `8.1.0` |
| `node`                             | `17.0.1` |

**Comment by bradzacher** [comment]
https://github.com/typescript-eslint/typescript-eslint/issues/1824
Please use the search next time
